Cargo.lock

```diff
@@ -12717,6 +12746,19 @@ dependencies = [
  "syn",
 ]
 
+[[package]]
+name = "xcm-transactor-primitives"
+version = "0.1.0"
+source = "git+https://github.com/integritee-network/pallets.git?branch=coax1d-test-transactor-branch#f4f570373e560efc692574bd728367fd30654a04"
+dependencies = [
+ "common-primitives 0.1.0 (git+https://github.com/integritee-network/pallets.git?branch=coax1d-test-transactor-branch)",
+ "cumulus-primitives-core",
+ "frame-support",
+ "parity-scale-codec",
+ "sp-std",
+ "xcm",
+]
+
```

Comment:
Nice, now we can be sure that the duplicate substrate deps from the subxt dev-deps don't end up in the other crates. :)